docs/primer/how.md

```diff
@@ -108,13 +108,91 @@ Some schema fields refer to [codelists](../schema/codelists), to limit and stand
 
 In addition to the default format of JSON, you can convert and publish your OCDS data as Excel and CSV (comma-separated values) files.
 
-```{admonition} Publication formats: JSON and Excel/CSV
+````{admonition} Publication formats: JSON and Excel/CSV
 :class: hint
 
 While the OCDS schema is described using JSON Schema, OCDS data can be converted from its JSON format to tabular formats such as CSV files or spreadsheets. JSON is favored by developers because it uses human-readable text to exchange complex information, such as nested objects. It can contain large volumes of information and is particularly good at handling one-to-many relationships (such as multiple bids per tender notice).
 
 Tabular formats, such as CSV (or comma separated values) are commonly used in spreadsheets applications and other analysis tools. Many people are comfortable working with spreadsheets using tools like Excel. While JSON is the default format, a good publication will publish tabular formats as well both so that more users’ needs can be satisfied.
+
+**Example**
+
+Ebonyi State in Nigeria publish OCDS data in JSON and tabular format. The JSON data is flattened into a series of CSV files which are combined into a single Excel spreadsheet. 
+
+The following example does not contain all existing fields, and not all of the included fields are required. It has been edited for length and consistency, and is not identical to the data available on the Ebonyi State e-Procurement platform.
```

Comment:
I agree that it should be implicit in the framing that something is an example and therefore not necessarily identical. I think it's worth including a disclaimer though to avoid (a probably unlikely tbf circumstance) of someone from e.g. Ebonyi State seeing this and feeling we've somehow misrepresented them by changing their data but not explicitly saying that's what we've done.

Reply:
re-reading and yes I agree that a generic disclaimer on the docs landing page would work, @jpmckinney what do you think?

Reply:
Yes, we can put a simple statement somewhere about the examples being edited for clarity.

@yolile Do we have preferences re: which publishers to take examples from? I'm not sure about the longevity of some publishers (and some publishers have very small or limited datasets). e.g. any examples in the Primer should be best-in-class in case anyone is inspired to look them up elsewhere.

Beginning to wonder if this issue is more trouble than it's worth.

Reply:
As far as I know, all the OCDS publishers have more or less some data issues, either quality, modeling, structure, longevity, access, etc. That is why I wasn't sure about referencing real examples in the first place, so I don't have a strong preference for any of them

Reply:
@jpmckinney I've done that in https://github.com/open-contracting/standard/issues/1449#issuecomment-1849353843

Reply:
Resolving as this PR no longer adds examples from real publishers.